src/popup/components/ace-editor/ace-editor.scss

```
@@ -0,0 +1,29 @@
+.ace-host {
+	width: 100%;
+	height: 100%;
+	border-radius: 10px;
+	overflow: hidden;
+	font-size: 15px;
+}
+
+:deep(.ace_editor),
+:deep(.ace_print-margin) {
+	background-color: var(--onSurface) !important;
+	font-family: 'Fira Code', 'Consolas', 'Source Code Pro', monospace;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)